### PR TITLE
Added SD template new 2

### PR DESCRIPTION
**Pull request by AlexanderHodicke**
New: 2

### CODE REVIEW

File: others/customer-documentation/solution-definition-mandatory/files/solution-definition_mandatory.md
Comment (oheimburger) [inline others/customer-documentation/solution-definition-mandatory/files/solution-definition_mandatory.md:1]: - Table in Mandatory Security Best Practices is broken. Please review.
- Naming Convention should be a chapter of it's own. It is not security related nor part of any architecture blueprint.
- Why a chapter for "Security and Identity Management" and "Mandatory Security Best Practices"? "Mandatory Security Best Practices" is meant to replace "Security and Identity Management"

File: others/customer-documentation/solution-definition-complete/files/solution-definition.md
Comment (oheimburger) [inline others/customer-documentation/solution-definition-complete/files/solution-definition.md:1]: - Table in Mandatory Security Best Practices is broken. Please review.
- Naming Convention should be a chapter of it's own. It is not security related nor part of any architecture blueprint.
- Why a chapter for "Security and Identity Management" and "Mandatory Security Best Practices"? "Mandatory Security Best Practices" is meant to replace "Security and Identity Management"

**Review by oheimburger — CHANGES_REQUESTED** [review]

File: others/customer-documentation/solution-definition-complete/files/solution-definition.md
Comment (AlexanderHodicke) [reply others/customer-documentation/solution-definition-complete/files/solution-definition.md:1]: - Done
- We agreed to leave it in LZ for now
- We will review this in a future iteration 

All done

File: others/customer-documentation/solution-definition-mandatory/files/solution-definition_mandatory.md
Comment (AlexanderHodicke) [reply others/customer-documentation/solution-definition-mandatory/files/solution-definition_mandatory.md:1]: Same as above

**Review by oheimburger — APPROVED** [review]
Assuming changes will applied in the next iteration.
Approved

**Pull request merged**